web-common-marlowe/src/Marlowe/Semantics.purs

```diff
@@ -638,15 +639,15 @@ instance hasArgsObservation :: Args Observation where
 validInterval :: SlotInterval -> Boolean
 validInterval (SlotInterval from to) = from <= to
 
-above :: Slot -> SlotInterval -> Boolean
+above :: POSIXTime -> SlotInterval -> Boolean
```

Comment:
```
[1/1 NoInstanceFound] web-common-marlowe/src/Marlowe/Semantics.purs:640:40

  640  validInterval (SlotInterval from to) = from <= to
                                              ^^^^^^^^^^

  No type class instance was found for

    Data.Ord.Ord POSIXTime

  while applying a function lessThanOrEq
    of type Ord t0 => t0 -> t0 -> Boolean
    to argument from
  while inferring the type of lessThanOrEq from
  in value declaration validInterval

  where t0 is an unknown type

           Src   Lib     All
Warnings   0     0/366   0/366
Errors     1     0       1
```

Reply:
That will disappear after I rebase on latest sprint-50